pkg/codegen/nodejs/gen.go

```diff
@@ -1181,8 +1183,13 @@ func (mod *modContext) genConfig(w io.Writer, variables []*schema.Property) erro
 			configFetch += " || " + v
```

Comment:
Unrelated to your change, but just wanted to point it out while looking over the code here.

The comment above this line says:

```typescript
// Note: this logic isn't quite correct, but already exists in all of the TF-based providers.
// Specifically, this doesn't work right if the first value is set to false but the default value
// is true.
```

We should be able to address this by using the `??` operator instead of `||`. Maybe worth considering as a separate change?

Reply:
Yeah I didn't mess with it because I figured the note was there for a reason but I can just fix it here.